src/modules/components/ModalWindow.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box } from "./Box";
+import { Button } from "./Button";
+
+
+const zIndex = {
+    modal: 1000,
+}
+
+export type ModalWindowProps = React.PropsWithChildren<{
+    isOpen: boolean, 
+    onClose: (event: React.MouseEvent<Element, MouseEvent>) => void,
+}>
+
+export const ModalWindow = ({ isOpen, onClose, children }: ModalWindowProps) => {
+
+    if (!isOpen) {
+        return null;
+    }
+
+    return (
+        <Box  
```

Comment:
`onClose` on background click would be nice, but perhaps not until there's a way to reopen modals for same offer/answer (or simplified 'conection' modal once we have that)
As a generic component though, for sure